source/_data/glossary.yml

```diff
@@ -104,6 +107,13 @@
     configuration-specific settings.
   aliases:
     - blueprints
+- term: Button
+  definition: >-
+    A button entity can fire an event / trigger an action towards a device or
+    service but remains stateless from the Home Assistant perspective.
```

Comment:
```suggestion
    A button entity can fire an event, or trigger an action towards a device or
    service. It can be compared to a physical switch or push button. The button entity does not have a state like `on` or `off`, but keeps the timestamp of when it was last pressed in the Home Assistant UI or via an action.
```